colearn_examples/ml_interface/pytorch_mnist_diffpriv.py

```diff
@@ -47,10 +47,10 @@
 vote_batches = 2
 
 # Differential Privacy parameters
-sample_size = 3300
-alphas = list(range(2, 32))
 noise_multiplier = 1.3
```

Comment:
I think the noise_multiplier is not used now at all.

Reply:
used at line 130 https://github.com/fetchai/colearn/pull/200/files/5603421d4f1273a7be98b1d133e449775a3f76b4#diff-4b45d33527b9a2d21975c6377934f51708553cc6fc90b2739c76582e34203550R130